### ISSUE TITLE
Extent encoding needs to be consolidated

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-bipm#62]
In the relaton model, the `locality` elements within an `extent` are hierarchical, and each `extent` is a separate span of the document.

So in BIPM Metrologia 8 1 32, the article is in volume 8, series 1, pages 32-36. That is a *single* extent element. If there are multiple extent elements, that would be describing a single article published in two separate volumes, each with their own pages.

So volume 8, series 1, pages 32-36 should be encoded as:

````
  <extent>
    <locality type="volume">
      <referenceFrom>8</referenceFrom>
    </locality>
    <locality type="issue">
      <referenceFrom>1</referenceFrom>
    </locality>
    <locality type="page">
      <referenceFrom>32</referenceFrom>
      <referenceTo>36</referenceTo>
    </locality>
  </extent>
````

It is currently encoded as:

````
 <extent>
    <locality type="volume">
      <referenceFrom>8</referenceFrom>
    </locality>
  </extent>
  <extent>
    <locality type="issue">
      <referenceFrom>1</referenceFrom>
    </locality>
  </extent>
  <extent>
    <locality type="page">
      <referenceFrom>32</referenceFrom>
      <referenceTo>36</referenceTo>
    </locality>
  </extent>
````

That is incorrect: it is interpreted as "volume 8, and (separately) issue 1, and (separately) pp. 32-36". When that is rendered, it is rendered with ";" separating what it thinks are three completely separate extents.

This presumably needs to be extended to all Metrologia publications.

**Comment by opoudjis** [comment]
Needed for rendering issue in https://github.com/metanorma/bipm-si-brochure/issues/225

**Comment by andrew2net** [comment]
fixed in relaton v1.20.0
